src/data/faq_de.json

```
@@ -146,6 +146,23 @@
                             "Unter dem Reiter Zertifikate kann die App nur '<b>EU Digitale</b> COVID-Zertifikate' scannen.",
                             "Siehe auch folgenden Eintrag zu <a href='#vac_cert_invalid' target='_self' rel='noopener'>ungültigen Impfzertifikaten</a>."
                          ]
+                    },
+                    {
+                        "title": "Wie lange ist mein Zertifikat gültig?​",
+                        "anchor": "eu_dcc_validity",
+                        "active": true,
+                        "textblock": [
+                            "Der folgende Text bezieht sich auf in Deutschland ausgestellte digitale COVID Zertifikate der EU, d.h. Impf-, Test-, und Genesenenzertifikate. Die angegeben Zahlen sind der aktuelle Stand vom Juli 2021.",

```

- Non breaking space between d. & h. (See https://de.wiktionary.org/wiki/d._h.)
- "vom Juli 2021" -> "von Juli 2021"

```suggestion
                            "Der folgende Text bezieht sich auf in Deutschland ausgestellte digitale COVID Zertifikate der EU, d.&nbsp;h. Impf-, Test-, und Genesenenzertifikate. Die angegeben Zahlen sind der aktuelle Stand von Juli 2021.",
```